roles/chameleon_usage/templates/chameleon-usage-extract-chi-data.j2

```diff
@@ -4,4 +4,4 @@ docker run --rm --net=host \
     -v "{{ backup_dir }}:{{ backup_dir }}" \
     -v "{{ chameleon_usage_config_dir }}/my.cnf:/etc/mysql/my.cnf" \
     {{ chameleon_usage_docker_image }} \
-    extract_data --site {{chameleon_site_name}} --backup-location {{ backup_dir }} --mysql-conf /etc/mysql/my.cnf 2>&1
+    extract_chi_data --site {{chameleon_site_name}} --backup-location {{ backup_dir }} --mysql-conf /etc/mysql/my.cnf 2>&1
```

Comment:
You could reuse this whole thing if the name of the function to run was also parameterized. What if we did that? Then you wouldn't have to create the separate KVM merge; you could simply set `usage_extract_command: extract_chi_data` versus `usage_extract_command: extract_kvm_data`.